plugins/rts54hub/rts54hub.quirk

```diff
@@ -0,0 +1,23 @@
+# RTS54 HS
+[DeviceInstanceId=USB\VID_0BDA&PID_5423]
+Plugin = rts54hub
+FirmwareSizeMin = 0x10000
+FirmwareSizeMax = 0x10000
+
+# RTS54 SS
+[DeviceInstanceId=USB\VID_0BDA&PID_0423]
+Plugin = rts54hub
+FirmwareSizeMin = 0x10000
+FirmwareSizeMax = 0x10000
+
+# Custom
+[DeviceInstanceId=USB\VID_0BDA&PID_5420]
+Plugin = rts54hub
+FirmwareSizeMin = 0x10000
+FirmwareSizeMax = 0x10000
```

Comment:
@AnyProblem do we want to include these other three entries?

Reply:
@hughsie  our fw size of 0x0bda 0x542x series from 64k to 256k depend on what kind of feature enable and IC type

Reply:
Not necessarily, our Hub flash is external SPI flash so it is depend on Project(OEM) requirement, not every project need to change VID and PID

Reply:
Given that it might be better for the reference implementation to max out at 256k too for `FirmwareSizeMax`.  Individual devices after they're known to have different VID/PID in their own quirks and using a particular SPI part in the design can be changed.

Reply:
This at least prevents potential problems if a project keeps the "reference" VID/PID.